origin/blobclient/cluster_client.go

```diff
@@ -108,6 +110,15 @@ func (c *clusterClient) defaultPollBackOff() backoff.BackOff {
 	}
 }
 
+func (c *clusterClient) CheckReadiness() error {
+	clients, err := c.resolver.Resolve(backend.ReadinessCheckDigest)
```

Comment:
please throw an error if list of clients resolved is empty as a sanity check.

Reply:
Resolve guarantees a non-empty list of clients. I made sure to go through the code to check and edited the comment.